README.md

```diff
@@ -146,6 +147,17 @@ Both `BaseCNN` and `BaseLSTM` also have prediction explanation mechanisms that c
 If you have ideas on how to improve the framework to assess text conversation for constructive and inclusive
 communication, we welcome your contributions!
 
+### Feature Vector creation 
+`featureVector.py` creates feature vector based on the `rawdatafile` and optionally `words` file. Default features 
+include sentiment and code blocks. `Words` file contains words important in measuring inclusiveness and 
+constructiveness. This functionality could be used instead of manual annotation.
+
+```python
+python ./ml-conversational-analytic-tool/featureVector.py <rawdatafile> --words <words_filename> --name <output_filename>
+```
+- `words` (optional) path to the words file
+- `name`  (optional) name of the output file.
+   
```

Comment:
I would keep this still under the header "Build Dataset" but outside of "Annotate" and also update the table of content (line 22)